### PR TITLE
Explain how to sync reporting

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 502b49c |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-406.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by vivekschauhan — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connected_agent_common_reference/traceability_usage.md [hunk 1]
```diff
@@ -19,10 +19,17 @@ Learn how to install and set up the Traceability Agent, using either the online
...
-* **Transaction** event: reports the transaction summary (API name, duration, status), and the transaction details (request/response headers from the frontend and backend of the API). This feature is optional and not required to report Gateway usage.
+* **Transaction Metrics** event: reports the total number of API calls during a period of time. This feature cannot be inactivated.  The Traceability Agent sends the Transaction Metrics to the database storage for display on Business Insights -> API Health (default is 15 minutes).  For more information on how to see an aggregate view of all Transactions, refer to [Business Insights](https://docs.axway.com/bundle/amplify-central/page/docs/get_actionable_insights/business_insights/index.html)
+* **Platform Usage** event: reports the total number of API calls during a period of time. This feature cannot be inactivated.  The Traceability Agent sends the Platform Usage counts to the database storage for display on Organization -> Usage (default is daily).  To clarify, the Platform Usage for a day is normally sent after a day has passed.  This would make Platform Usage a lagging indicatore compared to API Health by at worst a whole day.
```
Comment (eric34) [inline content/en/docs/connect_manage_environ/connected_agent_common_reference/traceability_usage.md:25]: "lagging indicatore" :)

**Review by vivekschauhan — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connected_agent_common_reference/traceability_usage.md [hunk 2]
```diff
@@ -105,30 +112,49 @@ After all files are copied, start Traceability Agent: `./traceability_agent ./ta
 
 For more information regarding agents' installation, see [Axway Gateway agents](/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli), [AWS Gateway agents](/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-with-amplify-cli), [Azure Gateway agent](/docs/connect_manage_environ/connect_azure_gateway/deploy-your-agents-with-amplify-cli) and [Istio Gateway agents](/docs/connect_manage_environ/mesh_management/deploy-your-agents-with-the-axway-cli).
 
-### Reporting Gateway usage event - automatic reporting for online mode
+### Reporting Gateway Transaction event - automatic reporting for online mode
```
Comment (vivekschauhan) [inline content/en/docs/connect_manage_environ/connected_agent_common_reference/traceability_usage.md:115]: This should be Transaction Metrics event, Transaction events are sent as the agent sees them

**Review by robertengaxway — APPROVED** [review]
Approved

File: content/en/docs/connect_manage_environ/connected_agent_common_reference/traceability_usage.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/connect_manage_environ/connected_agent_common_reference/traceability_usage.md]: fixed the typo

**Pull request merged**